api/src/Entity/User.php

```diff
@@ -194,6 +223,10 @@ public function getPassword(): ?string {
         return $this->password;
     }
 
+    public function ownsCamps(): bool {
```

Comment:
Wird hier absichtlich die Reihenfolge der Methode geändert?

Reply:
Ja, vormals hatte ich die zuunterst, und in diesem PR habe ich sie über die Boilerplate Collection Getters, Adders und Removers verschoben, weil die viel Platz wegnehmen und ich die oft einfach überlese. Wenn du einen anderen Vorschlag für die Reihenfolge von Methoden hast wäre ich offen.